### DISCUSSION TITLE
Angular Build/Deploy Using Development Environment

### DISCUSSION THREAD

**Discussion by UrzaMTG** [vercel/community#3465] [Help]
### Summary

I am deploying an Angular application to Vercel through Github integration. I have a singular development environment that changes the title on the page to include "-dev" so that I know it is using the development configuration of build, via angular.json. I have tried a bunch of different ways of forcing Vercel to use my production build, but it is not working. I have overridden the build command to specify "-c production" but I never see it show that change in the deployment log. I have tried changing the output directory. I have tried clearing the Development Command. I cannot get this to use my production settings and I need help.

Repo: https://github.com/UrzaMTG/twirch

### Example

https://twirch.vercel.app/

### Steps to Reproduce

Whenever I build and redeploy, no matter what I change, I see "-dev" in the one section of my page where that title is referenced.

**Comment by amyegan** [comment]
Hey @UrzaMTG. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better view of what's happening.

**Reply by UrzaMTG** [reply]
I added the repo to the question, it is at https://github.com/UrzaMTG/twirch
